src/current-user.js

```diff
@@ -434,6 +434,13 @@ export class CurrentUser {
   subscribeToRoom({ roomId, hooks = {}, messageLimit, serverInstance } = {}) {
     typeCheck("roomId", "string", roomId)
     typeCheckObj("hooks", "function", hooks)
+    if (!serverInstance && hooks.onMessageDeleted) {
+      // v2 does not send message_deleted events
+      // eslint-disable-next-line no-console
+      console.warn(
```

Comment:
We have access to `this.logger` so no need to use `console`
```suggestion
      this.logger.warn(
```